README.md

```diff
@@ -21,11 +21,12 @@ reply to other students’ questions.
 
 This project uses the following tools:
 
-- [YouTube Data API]() for generating transcripts
-- [YouTube iFrame API]() for displaying video
-- [AppEngine]() for deploying
-
-*TODO: Add urls and more APIs.*
+- [AppEngine on Google Cloud](https://cloud.google.com/appengine) for deploying
+- [Java Servlets](https://docs.oracle.com/javaee/5/tutorial/doc/bnafe.html) for back-end
+- [React](https://reactjs.org/) for front-end
```

Comment:
Are we for sure using React? Also, should we include Bootstrap?

Reply:
If we decide on using React:
```suggestion
- [React](https://reactjs.org/) and [Bootstrap](https://react-bootstrap.github.io/) for front-end
```

Reply:
Remove React
```suggestion
```